lib/genome/groupers/drug_grouper.rb

```diff
@@ -1,154 +1,149 @@
 module Genome
   module Groupers
     class DrugGrouper
-      @alt_to_pubchem = Hash.new() {|hash, key| hash[key] = []}
-      @alt_to_other = Hash.new() {|hash, key| hash[key] = []}
-      @alt_to_pubchem_cid = Hash.new() {|hash, key| hash[key] = []}
 
-      def self.run
-        ActiveRecord::Base.transaction do
-          puts 'reset groups'
-          reset
-          puts 'preload'
-          preload
-          puts 'create groups'
-          create_groups
-          puts 'add members'
-          add_members
-          puts 'add drug aliases'
-          add_aliases
-          puts 'add drug attributes'
-          add_attributes
+      def run
+        begin
+          newly_added_claims_count = 0
+          drug_claims_not_in_groups.in_groups_of(1000, fill_with=false) do |claims|
```

Comment:
I do think that this is doing what I expect (based on documentation from https://apidock.com/rails/Array/in_groups_of), although I agree that at first glance it seems wrong, and should instead by something like `fill_with: false`. Sample IRB below. 

```ruby
[6] pry(main)> >> a = [1, 3, 5, 7, 9]
=> [1, 3, 5, 7, 9]
[7] pry(main)> >> a.in_groups_of(2)
=> [[1, 3], [5, 7], [9, nil]]
[8] pry(main)> >> a.in_groups_of(2, fill_with=false)
=> [[1, 3], [5, 7], [9]]
[9] pry(main)> >> a.in_groups_of(2, fill_with: false)
=> [[1, 3], [5, 7], [9, {:fill_with=>false}]]
[10] pry(main)> >> a.in_groups_of(2, false)
=> [[1, 3], [5, 7], [9]]
```